src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -267,6 +274,24 @@ export class PatientViewPageStore {
         this.openResourceTabMap.set(resourceId, open);
     }
 
+    @autobind
```

Comment:
i think these bound actions should really be on view, not on store.

Reply:
agreed